client/ayon_houdini/api/plugin.py

```diff
@@ -142,6 +152,17 @@ def create(self, product_name, instance_data, pre_create_data):
                 product_name,
                 instance_data,
                 self)
+            
+            if self.enable_staging_path_management:
+                staging_dir = self.get_staging_dir(instance)
+
+                with set_pwd(instance_node):
+                    if self.expand_staging_dir:
+                        # Expand vars only without expanding expressions to keep dynamic link to ROP parameters.
+                        staging_dir = expand_houdini_string(staging_dir)
```

Comment:
We could swap this, right?
```suggestion
                if self.expand_staging_dir:
                    with set_pwd(instance_node):
                        # Expand vars only without expanding expressions to keep dynamic link to ROP parameters.
                        staging_dir = expand_houdini_string(staging_dir)
```

Reply:
good catch. I tweaked it a bit.
https://github.com/ynput/ayon-houdini/pull/200/commits/a3123a75a1d22872642d5f28a2cafc6f230e1afa